pyproject.toml

```diff
@@ -36,7 +36,7 @@ classifiers = [
 
 
 [tool.poetry.dependencies]
-python = ">=3.7,<3.10"
+python = ">=3.7,<3.12"
```

Comment:
```suggestion
python = ">=3.8,<3.12"
```

Reply:
Done

Reply:
This will also need update of the categories in pyproject.toml, and of the workflows in GA in order to get tests to pass (since 3.7 is now excluded) but also to test new versions we weren't previously testing.